_search-plugins/sql/functions.md

```diff
@@ -19,12 +19,12 @@ The SQL plugin supports the following functions.
 Function | Specification | Example
 :--- | :--- | :---
 abs | `abs(number T) -> T` | `SELECT abs(0.5) FROM my-index LIMIT 1`
```

Comment:
@matthewryanwells the `FROM my-index LIMIT 1` throughout the file can be removed.  The SQL plug no longer requires that a `FROM` table is included, and the statement `SELECT abs(0.5)` is now valid.